packages/disputer/src/disputer.js

```diff
@@ -15,7 +15,7 @@ class Disputer {
    * @param {String} account Ethereum account from which to send txns.
    * @param {Object} empProps Contains EMP contract state data. Expected:
    *      { priceIdentifier: hex("ETH/BTC") }
-   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
+   * @param {Object} [disputerConfig] Contains fields with which constructor will attempt to override defaults.
```

Comment:
nit: not related to this line, but the title doesn't conform to conventional commit standards:
- `bug` should be changed to `fix`
- There should be a `:` after `fix(...)`.

Correcting those two, the title should be:
```
fix(monitor,disputer,liquidator bots): correctly pass in bot config objects into bots
```

Reply:
thanks for pointing this out. will follow the convention correctly in the future :)